blazor/diagram-classic/ports/ports.md

```diff
@@ -9,7 +9,7 @@ documentation: ug
 
 N> Syncfusion recommends using [Blazor Diagram Component](https://blazor.syncfusion.com/documentation/diagram/getting-started) which provides better performance than this diagram control. Blazor Diagram Component will be actively developed in the future.
 
-# Actions of port in Blazor Diagram Component
+# Actions of Port in Blazor Diagram Component
```

Comment:
where is page with redirect changes here?

Reply:
There was no redirect links so I hard to work on Site audit changes.